src/settings/StaffSlips/components/ViewSections/StaffSlipTemplateContentSection/StaffSlipTemplateContentSection.test.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { PreviewModal } from '@folio/stripes-template-editor';
+
+import StaffSlipTemplateContentSection from './StaffSlipTemplateContentSection';
+
+const mockParseWithInstructionsReturnValue = 'testParsedValue';
+const mockTokensReducerReturnValue = 'testReturnValue';
+
+jest.mock('html-to-react', () => ({
+  __esModule: true,
+  default: {
+    ProcessNodeDefinitions: jest.fn(() => ({
+      processDefaultNode: 'testDefaultNode',
+    })),
+  },
+  Parser: () => ({
+    parseWithInstructions: jest.fn(() => mockParseWithInstructionsReturnValue),
+  }),
+}));
+jest.mock('@folio/stripes-template-editor', () => ({
+  PreviewModal: jest.fn(({
+    'data-testid': testId,
+  }) => (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+    <div data-testid={testId} />
+  )),
+  tokensReducer: () => mockTokensReducerReturnValue,
+}));
+
+describe('StaffSlipTemplateContentSection', () => {
+  const testIds = {
+    staffSlipsPreviewButton: 'staffSlipsPreviewButton',
+    previewModal: 'previewModal',
+  };
+  const testInitialValues = {
+    name: 'testNameValue',
+    description: 'testDescriptionValue',
+    template: 'testTemplateValue',
+  };
+  const labelIds = {
+    staffSlipsPreview: 'ui-circulation.settings.staffSlips.preview',
+    staffSlipsViewPreviewLabel: 'ui-circulation.settings.staffSlips.view.previewLabel',
+  };
+  const getById = (id) => within(screen.getByTestId(id));
+
+  afterEach(() => {
+    PreviewModal.mockClear();
+  });
+
+  beforeEach(() => {
+    render(<StaffSlipTemplateContentSection staffSlip={testInitialValues} />);
+  });
+
+  it('render StaffSlipTemplateContentSection', () => {
+    expect(screen.getByText('ui-circulation.settings.staffSlips.display')).toBeVisible();
```

Comment:
Could you add `'ui-circulation.settings.staffSlips.display'` to `labelIds` object?